src/components/Header/HeaderContact.js

```diff
@@ -90,12 +90,16 @@ const styles = StyleSheet.create({
   }),
   text: {
     color: COLORS.black,
-    fontFamily: fonts.inter[600],
+    fontFamily: fonts.inter[500],
     fontSize: normalizeFontSize(14),
-    fontWeight: 'bold',
     padding: 10,
     textAlign: 'center'
   },
+  titleText: {
+    position: 'absolute',
+    width: '100%',
+    zIndex: -1
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `titleText` style object seems to be aimed at positioning the title text absolutely within its parent container. This could potentially impact the layout of any components using this style. Make sure this change does not cause any unexpected layout shifts.
```diff
+ titleText: {
+   position: 'absolute',
+   width: '100%',
+   zIndex: -1
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->